microsoft-edge/devtools-guide-chromium/whats-new/whats-new-archive.md

```diff
@@ -0,0 +1,34 @@
+---
+title: What's new in Microsoft Edge DevTools Archives
+description: New features and fixes in Microsoft Edge DevTools Archives
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/23/2023
+---
+# Archived What's New entries
```

Comment:
```suggestion
# What's New archive
```
possibly change to efficient short form, avoids having many words and then finally 'archive'; only 2 words and then 'archive'.
Capital N for best parsing.

Alternative titles:
- What's new archive
- What's New archive
- What's new in Microsoft Edge DevTools Archives
- Archived What's New entries